Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -518,7 +521,7 @@ def main():  # pragma: no cover
         elif command == "fetch-events":
             page_size = int(params.get("page_size", FETCH_EVENTS_PAGE_SIZE))
             limit = int(params.get("fetchLimit", 300000))
-            for events, offset, total_events_count, hashed_events_from_previous_run in fetch_events_command(  # noqa: B007
+            for events, offset, total_events_count, hashed_events_from_current_run in fetch_events_command(  # noqa: B007
```

Comment:
hashed_events_from_current_run is being set here overwriting what was set in line 459 - i generally think its a bad idea to re-set variables because it makes it a bit confusing, but not a must